api/emAnno/emAnnoService.ts

```diff
@@ -1,24 +1,29 @@
-import {AxiosResponse} from 'axios'
+import { AxiosResponse } from 'axios'
 import { http } from '../lib/http'
 import * as log4jui from '../lib/log4jui'
 import { EnhancedRequest, JUILogger } from '../lib/models'
 import { setHeaders } from '../lib/proxy'
-import { Annotation, Annotations } from './models'
+import { Annotation } from './models'
 
 const logger: JUILogger = log4jui.getLogger('em-anno-service')
 
 /**
  * Get Annotations
  *
  * @param annotationsPath
- * @param req
- * @returns {Promise<null>}
+ * @returns {Promise<AxiosResponse>}
  */
-export async function handleGet(annotationsPath: string, req: EnhancedRequest): Promise<Annotations> {
-    logger.info('getting annotations', annotationsPath)
-    const headers = setHeaders(req)
-    const response: AxiosResponse<Annotations> = await http.get(annotationsPath, { headers })
-    return response.data
+export async function handleGet(annotationsPath: string, req: EnhancedRequest): Promise<AxiosResponse> {
+
+    try {
+        logger.info('getting annotations', annotationsPath)
+        const headers = setHeaders(req)
+        const response = await http.get(annotationsPath, { headers })
+        return response
+    } catch (e) {
+        logger.error(e.message)
```

Comment:
we're again assuming properties to always be present on the error, as we learnt during the P1 issue. Also as we have an errorHandler you should probably pass the error to it via NextFunction if possible?